tests/InputFiles/GrMhd/GhValenciaDivClean/BinaryNeutronStar.yaml

```diff
@@ -241,7 +307,7 @@ EventsAndTriggers:
 
 InterpolationTargets:
   BondiSachsInterpolation:
-    LMax: 16
+    LMax: 8
```

Comment:
Maybe keep this one? 16 is what is typically used in SpEC and you've disabled the event above